typescript/sdk/src/contracts.ts

```diff
@@ -72,13 +69,11 @@ export abstract class AbacusContracts<
 }
 
 export type RouterAddresses = {
-  upgradeBeaconController: types.Address;
   abacusConnectionManager: types.Address;
-  router: ProxiedAddress;
+  router: types.Address;
```

Comment:
`AbacusContracts` already handles `types.Address | ProxiedAddress`
maybe we should do that here and everywhere for the addresses shapes

Reply:
I'm not entirely sure how exactly that would look?